Lena's attempt.Rmd

```diff
@@ -59,14 +63,15 @@ plot(score_ctree)
 ```
 
 Please interpret the tree, which two behaviors do you think the teacher should most closely pay attemtion to?
+//we choose node 7 and 9; namely behavior 1 0<hints<12 & prior_percent_correct <=0.62; behavior 2 hints>12
 
```

Comment:
Can you provide a more substantive explanation of the tree. What would you tell a teacher too look out for in their students?